samples/deployment/README.md

```diff
@@ -0,0 +1,35 @@
+# Forgeops Deployment
+
+### Prerequisites
+This guide assumes you have access to Kubernetes cluster and you are able to
```

Comment:
s/to Kubernetes cluster/to a Kubernetes cluster/

Reply:
By "able to do deployments," not sure what you mean. Is it "you have RBAC privileges to do deployments?" You might want to be a little more explicit about this in this comment.